spec/inputs/file_spec.rb

```diff
@@ -1,23 +1,14 @@
 # encoding: utf-8
 
-require "logstash/devutils/rspec/spec_helper"
+require "spec/spec_helper"
```

Comment:
This will conflict when you run the test next to all the other plugins test, this need to be a "require_relative"